README.md

```diff
@@ -103,7 +103,7 @@ depends on your service discovery registry. To see the flags available for your
 SD, run:
 
 ```console
-$ docker run turbinelabs/rotor:0.17.0 rotor <platform> --help
+$ docker run --entrypoint rotor turbinelabs/rotor:0.17.0 <platform> --help
```

Comment:
I think this can be simpler; with your entrypoint and arg-passing change, you can take advantage of the built-in `help` command in rotor, and just do:

`docker run turbinelabs/rotor:0.17.0 help <platform>`

Reply:
That won't work. The correct way to use the inbuilt ```help``` command would be:

```docker run -e "ROTOR_CMD=help" rotor-file-ep:latest <platform>```

Reply:
ah right. I think that's still a little simpler for folks, since it mirrors the way other commands are run.

The _other_ way we could do this would be to add an optional flag to the file collector to specify the path, in addition to reading it as an arg. That would allow us to say that all collectors are fully specifiable, which would let us keep the simpler calling semantics for help. I think I marginally prefer this approach, as it keeps the overall ergonomics simpler.

We can make that change internally.

Reply:
In that case, do you still need this PR?

Reply:
No I think we're good. Thank you for finding this!